L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc

```diff
@@ -51,6 +51,8 @@ void L1TGlobalProducer::fillDescriptions(edm::ConfigurationDescriptions& descrip
       ->setComment("InputTag for Calo Trigger EtSum (required parameter:  default value is invalid)");
   desc.add<edm::InputTag>("EtSumZdcInputTag", edm::InputTag(""))
       ->setComment("InputTag for ZDC EtSums Plus and Minus (required parameter:  default value is invalid)");
+  desc.add<edm::InputTag>("CICADAInputTag", edm::InputTag(""))
+      ->setComment("InputTag for CICADA Anomaly Detection (required parameter: default value is invalid)");
```

Comment:
So now that this PR and its backport are in 14_1 / 14_0, what value for this new parameter should be used at HLT level running the L1TGlobalProducer? ```hltGtStage2ObjectMap = cms.EDProducer( "L1TGlobalProducer",...```

Reply:
@Martin-Grunewald Do you want emulator or DAQ values?

Reply:
So in HLT where we run ```hltGtStage2Digis = cms.EDProducer( "L1TRawToDigi",...```
it would be ```hltGtStage2ObjectMap.CICADAInputTag = cms.InputTag("hltGtStage2Digis","CICADAScore"),```, correct?

Reply:
Can this discussion be documented in a [CMSHLT](https://its.cern.ch/jira/projects/CMSHLT/issues) Jira ticket please @aloeliger  ?

Reply:
https://its.cern.ch/jira/browse/CMSHLT-3280